Makefile.tt

```diff
@@ -0,0 +1,27 @@
+include .env
+
+.PHONY: dev env/setup env/teardown codebase codebase/fix
+
+dev:
+	make install-dependencies
+	make env/setup
+	./bin/dev
+
+env/setup:
+	./bin/envsetup.sh
+	rails db:prepare
+
+env/teardown:  # this command will delete data
+	./bin/envteardown.sh
+
+install-dependencies:
+	bundle install
+	yarn install
+
+codebase:
+	rubocop
+	yarn codebase
```

Comment:
do we have this `command` in the `package.json` file 🤔 - same as `codebase:fix` yet?

Reply:
yes, they are:

https://github.com/nimblehq/rails-templates/blob/feb0cbc2979c8a26a7e0c200870b407e0445bae0/.template/variants/web/package.json.rb#L31-L32